contrib/golang/filters/http/source/go/Makefile

```diff
@@ -0,0 +1,20 @@
+BUILD_IMAGE  = golang:1.18
+PROJECT_NAME = envoy-go-extension
+
+build-local: sync
+	go build \
+		-v \
+		--buildmode=c-shared \
+		-o libgolang.so \
+		.
+
+build:
+	docker run --rm -v $(shell pwd):/go/src/${PROJECT_NAME} -w /go/src/${PROJECT_NAME} ${BUILD_IMAGE} make build-local
+
+sync:
+	cp export.go pkg/api/export.txt
+
+clean:
+	@rm -rf libgolang.*
+
+.PHONY: sync build-local build
```

Comment:
What is this for? Shouldn't we be using bazel here?

Reply:
Done. use bazel instead of Makefile.